stages/org.osbuild.fix-bls

```diff
@@ -13,9 +13,22 @@ This happens because some boot loader config tools (e.g. grub2-mkrelpath)
 examine /proc/self/mountinfo to find the "real" path to /boot, and find the
 path to the osbuild tree - which won't be valid at boot time for this image.
 
+Additionally, if "options" is specified it will replace all "options", i.e.
+the kernel parameters, with the given value. Wrong values there can happen
```

Comment:
Didn't you mention that this could be avoided by instead dropping in the right /etc/kerne/cmdline file? Might not that be preferable?

Reply:
Yeah, indeed, I try to explain that in the comment: "I left the fix-bls changes in (now unused) because maybe they come in handy later. Can easily drop the commits though."

Reply:
Ah, right. I did read that, but didn't make the connection. I'd drop it. Just because fix-bls is something we should phase out, so should make it clear what kind of hacks we actually use.

Reply:
Done.